github/release_notes/renderer.py

```diff
@@ -216,20 +217,16 @@ def _header_suffix_reference(
             if should_generate_link:
                 reference_id_text = rls_note_obj.reference.identifier[0:12] # short commit hash
 
-        reference = '{reference_prefix}{ref_id}'.format(
-            reference_prefix=reference_prefix,
-            ref_id=reference_id_text,
-        )
-
+        reference = f'{reference_prefix}{reference_id_text}'
         if is_reference_auto_linked:
```

Comment:
nit: in general, it would make reviewing a lot easier if refactorings like this (there are also others) would be put into separate commits :-/